samples/kaspresso-sample/build.gradle.kts

```diff
@@ -53,7 +54,9 @@ dependencies {
     testImplementation(libs.androidXTestExtJunit)
     testImplementation(libs.robolectric)
 
-    debugImplementation(libs.androidXTestFragmentTesting)
+    debugImplementation(libs.androidXTestFragmentTesting) {
+        isTransitive = false
```

Comment:
why? if it necessary, please, add comment why.

Reply:
Latest compose libs bump in master broke sample tests. There was a library conflict, some methods haven't been found in runtime. This fixes the issue

Reply:
please add comment here, with error and error reason

Reply:
Done